fuse/data/datasets/dataset_default.py

```diff
@@ -43,6 +38,7 @@ def __init__(
         dynamic_pipeline: Optional[PipelineDefault] = None,
         cacher: Optional[SamplesCacher] = None,
         allow_uncached_sample_morphing: bool = False,
+        always_return_dict = True,
```

Comment:
@michalozeryflato , is it your change? Can we drop it?

Reply:
@itaijj can you explain why this is needed?